fireeyeetp_consts.py

```diff
@@ -25,7 +25,10 @@
 ERR_CODE_MSG = "Error code unavailable"
 ERR_MSG_UNAVAILABLE = "Error message unavailable. Please check the asset configuration and|or action parameters"
 PARSE_ERR_MSG = "Unable to parse the error message. Please check the asset configuration and|or action parameters"
-TYPE_ERR_MSG = "Error occurred while connecting to the Fireeye ETP Server. Please check the asset configuration and|or the action parameters"
+TYPE_ERR_MSG = "Error occurred while connecting to the Fireeye ETP Server. "
+TYPE_ERR_MSG += "Please check the asset configuration and|or the action parameters"
+ERR_ISO_FORMAT = "Date supplied in the modified_date field is not ISO8601 compliant. "
+ERR_ISO_FORMAT += "Please make sure it is a valid ISO8601 datetime stamp"
```

Comment:
nit: can we use a line continuation for these to avoid the variable name duplication

```suggestion
TYPE_ERR_MSG = "Error occurred while connecting to the Fireeye ETP Server. " \
    "Please check the asset configuration and|or the action parameters"
ERR_ISO_FORMAT = "Date supplied in the modified_date field is not ISO8601 compliant. " \
    "Please make sure it is a valid ISO8601 datetime stamp"
```